packages/commands/src/utils.mjs

```diff
@@ -2,23 +2,46 @@ import {
   sdk as AudiusSdk,
   DiscoveryNodeSelector,
   SolanaRelay,
-  Configuration
+  Configuration,
+  UserAuth
 } from '@audius/sdk'
 
+import { WalletManager, getPlatformCreateKey } from '@audius/hedgehog'
+
 import {
   Utils as AudiusUtils,
   libs as AudiusLibs
 } from '@audius/sdk-legacy/dist/libs.js'
 import { PublicKey } from '@solana/web3.js'
 
+import { LocalStorage } from 'node-localstorage'
+const localStorage = new LocalStorage('./local-storage')
+
+export const parseUserId = async (arg) => {
```

Comment:
Not something you changed, but is this parseUserId function mainly for parseSplWallet? I haven't seen the `@` / `#` prefixes for user ids before

Also noticed there is an unused import of parseUserId in unfollow.mjs

Reply:
Yeah this is just kind of a long-existing utility because some of the commands do hacky stuff.
Once we get everything on to SDK, this can go away, because we won't have any need to parse user ids to numbers.